dev/ProgressRing/ProgressRing.xaml

```diff
@@ -15,35 +14,26 @@
         <Setter Property="MinWidth" Value="16" />
         <Setter Property="IsTabStop" Value="False" />
         <Setter Property="Width" Value="32" />
+        <Setter Property="Maximum" Value="100" />
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="local:ProgressRing">
                     <Grid x:Name="LayoutRoot" Background="Transparent">
                         <VisualStateManager.VisualStateGroups>
                             <VisualStateGroup x:Name="CommonStates">
-                                <VisualState x:Name="Active">
-                                    <Storyboard>
-                                        <DoubleAnimation
-                                            Storyboard.TargetName="LayoutRoot"
-                                            Storyboard.TargetProperty="Opacity"
-                                            To="1"
-                                            Duration="0:0:0.15" />
-                                    </Storyboard>
-                                </VisualState>
                                 <VisualState x:Name="Inactive">
-                                    <Storyboard>
-                                        <DoubleAnimation
-                                            Storyboard.TargetName="LayoutRoot"
-                                            Storyboard.TargetProperty="Opacity"
-                                            To="0"
-                                            Duration="0" />
-                                    </Storyboard>
+                                    <VisualState.Setters>
+                                        <Setter Target="LayoutRoot.Opacity" Value="0" />
+                                    </VisualState.Setters>
                                 </VisualState>
+                                <VisualState x:Name="DeterminateActive"/>
```

Comment:
>DeterminateActive [](start = 53, length = 17)

Why are we adding this state now?

Reply:
Technically, we don't need a `DeterminateActive` state in xaml as we reduced AVP to just one. However, `Active` and `DeterminateActive` are still two different states and should be in the visual state manager should someone wish to re-template.

Reply:
intersting. I think that makes sense, @ranjeshj to double check.

Reply:
Yes. I think this makes sense. It would have been nice to match these to ProgressBar states but we can't do that without breaking compat. @MarissaMatt  can you please add this info to the spec.